### PR TITLE
Added userAccountControl attribute

### PR DESCRIPTION
**Pull request by bapthbrt**
Added userAccountControl attribute to correctly create the computer account in Active Directory.
This commit closes zakird/pyad#100

Bonus: Added tail '$' at the end of sAMAccountName attribute as a best practice.

### CODE REVIEW

File: pyad/adcontainer.py [hunk 1]
```diff
@@ -77,7 +77,11 @@ def create_computer(self, name, enable=True,optional_attributes={}):
...
             obj = self.__create_object('computer', name)
-            obj.Put('sAMAccountName', name)
+            obj.Put('sAMAccountName', name + '$')
+            if enable:
+                obj.Put('userAccountControl', 4128)
+            else:
+                obj.Put('userAccountControl', 4130)
             obj.SetInfo()
             pyadobj = ADComputer.from_com_object(obj)
             if enable:
```
Comment (zakird) [inline pyad/adcontainer.py]: Seems like this needs to change if we're changing how enable works above. is there any rason that those values shoouldn't just be set in`.enable()`?

Comment (bapthbrt) [reply pyad/adcontainer.py]: It has to be set at creation otherwise you'll end with a user account instead of a computer one (see #100).

Comment (zakird) [reply pyad/adcontainer.py]: Makes sense. DO we still need to call `.enable()`?

Comment (bapthbrt) [reply pyad/adcontainer.py]: Yes, by default the object is disabled.

**Pull request merged**